engine/gl/gl_vidcommon.c

```diff
@@ -282,11 +282,6 @@ FTEPFNGLACTIVESTENCILFACEEXTPROC qglActiveStencilFaceEXT;
 
 
 #define GLchar char
-#ifdef NACL
-#undef DEBUG
-#elif defined(_DEBUG) && !defined(DEBUG)
-#define DEBUG
```

Comment:
this looks unintended

Reply:
You're right, this is wrong. Let me fix this

Reply:
Thank you, IRC Spoike said it's OK to merge!